modules/server_new/src/main/scala/observe/server/ObserveEngineImpl.scala

```diff
@@ -1306,7 +1274,7 @@ private class ObserveEngineImpl[F[_]: Async: Logger](
 //                    if q.status(st) =!= BatchExecState.Completed =>
 //                  runNextsInQueue(qid, observer, user, clid, freed).flatMap { l =>
 //                    Handle.fromStream(
-//                      Stream.emit[F, EventType[F]](
+//                      Stream.emit[F, Event[F]](
 //                        Event.modifyState[F, EngineState[F], SeqEvent](
 //                          executeEngine.pure(SequencesStart(l))
 //                        )
```

Comment:
And these lines?

Reply:
I left all the lines that were commented assuming that's code from seqexec that we will use again at some point. Should I remove them?